src/style.css

```diff
@@ -0,0 +1,145 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
+  display: flex;
+  justify-content: center;
+  color: #060b35;
+  margin: 0;
+}
+
+.body__body {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.body__item-picture {
+  display: flex;
+  padding: 32px 19px 0;
+  margin: 0;
+}
+
+.body__imac-image {
+  width: 160px;
+  height: 134px;
+}
+
+.body__item-name {
+  font-family: Roboto , sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  padding: 40px 16px 0;
+}
+
+.body__item-code {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
`font-style: normal;
  font-weight: 400;` is a default value, no need to set it